trans.py

```diff
@@ -2,6 +2,25 @@
 import struct
 import logging
 
+def read_file():
+    '''To read the filename and the number of files that have to be interpreted from infos.txt'''
+    with open("infos.txt", "r", encoding="UTF-8") as info_read:
+        raw_file = info_read.readlines()
+        read_data = []
+        # Getting the name of the files so the file manager can find it
+        for line in raw_file:
+            read_data.append(line.strip())
+        in_pos = read_data.index("filestoread")
+        fin_pos= read_data.index("endfilestoread")
+        name_files = []
+        i = 0
+        for counter in range(in_pos+1, fin_pos):
+            name_files.append(read_data[counter])
+            i += 1
+        if i:
+            return name_files, i #returns string list of files and how many of them to read
+        return 'No groups to read'
```

Comment:
You're returning different things here (files or string) mixing datatypes like this can lead to trouble later

Reply:
How do I fix this if I need that counter for other ops? Making a new function from scratch?